map/styled_point.cpp

```diff
@@ -0,0 +1,56 @@
+#include "map/styled_point.hpp"
+
+
+graphics::DisplayList * StyledPoint::GetDisplayList(UserMarkDLCache * cache) const
+{
+  UserMarkDLCache::Key key = GetStyle().empty() ? GetContainer()->GetDefaultKey()
+                                                : UserMarkDLCache::Key(GetStyle(),
+                                                                       graphics::EPosAbove,
+                                                                       GetContainer()->GetDepth());
```

Comment:
UserMarkContainer const * container = GetContainer();
and reuse container variable in two places here.